android-agent/src/main/kotlin/io/opentelemetry/android/agent/OpenTelemetryRumInitializer.kt

```diff
@@ -48,9 +51,21 @@ object OpenTelemetryRumInitializer {
                 endpointHeaders,
             ),
         rumConfig: OtelRumConfig = OtelRumConfig(),
-    ): OpenTelemetryRum =
-        OpenTelemetryRum
+        sessionConfig: SessionConfig = SessionConfig(),
+    ): OpenTelemetryRum {
+        // The session manager will be a SessionProvider implementation living in the agent where the agent will
+        // have full control of what a session is and will also make it observable and configurable.
+        val sessionManager =
```

Comment:
The agent will initialize and set our opinionated SessionProvider impl during its initialization.